src/zui/ZUICleanHtml.tsx

```diff
@@ -4,15 +4,23 @@ import { Box, BoxProps } from '@mui/material';
 import { useMemo } from 'react';
 
 interface ZUICleanHtmlProps {
-  dirtyHtml: string;
+  allowTargetBlank?: boolean;
   BoxProps?: BoxProps;
+  dirtyHtml: string;
 }
 
 const ZUICleanHtml = ({
+  allowTargetBlank = false,
   BoxProps,
   dirtyHtml,
 }: ZUICleanHtmlProps): JSX.Element => {
-  const cleanHtml = useMemo(() => DOMPurify.sanitize(dirtyHtml), [dirtyHtml]);
+  const cleanHtml = useMemo(
+    () =>
+      DOMPurify.sanitize(dirtyHtml, {
+        ADD_ATTR: allowTargetBlank ? ['target', 'rel'] : [],
+      }),
+    [dirtyHtml]
+  );
```

Comment:
This is only needed because we are passing modified HTML, instead of modifying the DOM after rendering (see separate comment).